src/setup/node.rs

```diff
@@ -202,17 +244,16 @@ impl Drop for Node {
     }
 }
 
-pub fn build_stateful_path() -> io::Result<PathBuf> {
+fn get_stateful_node_path() -> io::Result<PathBuf> {
```

Comment:
good call - we shouldn't use "build" for anything that doesn't employ the builder pattern :+1: